nodes/src/nodes/llm_mistral/mistral.py

```
@@ -37,7 +37,7 @@
 requirements = os.path.dirname(os.path.realpath(__file__)) + '/requirements.txt'
 depends(requirements)
 
-from typing import Any, Dict, Tuple
+from typing import Any, Callable, Dict, Tuple
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)